nltk/test/unit/test_seekable_unicode_stream_reader.py

```diff
@@ -1,69 +1,37 @@
 # -*- coding: utf-8 -*-
-"""
-The following test performs a random series of reads, seeks, and
-tells, and checks that the results are consistent.
-"""
-import random
-import functools
 from io import BytesIO
+import os
+import pytest
+
 from nltk.corpus.reader import SeekableUnicodeStreamReader
 
 
-def check_reader(unicode_string, encoding, n=1000):
+def check_reader(unicode_string, encoding):
```

Comment:
this was kinda a weird test, so i rewrote it. things i wasn't crazy about:

- Basing a test off randomization inherently feels odd?
- On average, it would loop about 6k times per string.
- the physical operations themselves were a bit hard to understand. Like, when ` op == 't'` it would just `.tell()` but not do anything with that information. It takes a while to understand the other `op` conditions as well

it feels better to just read in the information in a few ways, and check it against the original string